### PR TITLE
dxFileManager: describe toolbar items

### PR DESCRIPTION
**Pull request by AnnaKondratova**

### CODE REVIEW

File: api-reference/_hidden/dxFileManagerToolbar/items/items.md [hunk 1]
```diff
@@ -11,8 +11,27 @@ Configures toolbar items' settings.
...
+- **'refresh'** - Refreshes the file manager content and shows the progress panel.
+- **'download'** - Downloads a file.
+- **'move'** - Moves files and directories.
+- **'copy'** - Copies files and directories.
+- **'rename'** - Renames files and directories.
+- **'delete'** - Deletes files and directories.
+- **'switchView'** - Switches between the 'Details' and 'Thumbnails' file system representation modes.
+- **'clearSelection'** - Clears selection from files and directories in the Item View area.
+
+To add a predefined item to the toolbar, specify its [name](/Documentation/ApiReference/UI_Widgets/dxFileManager/Configuration/toolbar/items/#name) and optional settings ('visible', 'location', 'locateInMenu', 'text', 'icon', 'disabled') to the [items](/Documentation/ApiReference/UI_Widgets/dxFileManager/Configuration/toolbar/items/) (for all items) or [fileSelectionItems](/Documentation/ApiReference/UI_Widgets/dxFileManager/Configuration/toolbar/fileSelectionItems/) (for selected items) collections.
```
Comment (dirk-pieterse) [inline api-reference/_hidden/dxFileManagerToolbar/items/items.md]: How can you "specify something to" something else? Why do you use this preposition?

File: api-reference/_hidden/dxFileManagerToolbar/items/items.md [hunk 2]
```diff
@@ -23,30 +42,93 @@ The **FileManager** widget allows you to add default and create custom toolbar i
...
+        });
+    });
+
+---
+
+**Custom Items**
+
+To add a custom toolbar item, specify its [text](/Documentation/ApiReference/UI_Widgets/dxFileManager/Configuration/toolbar/items/#text) and optional settings (for example, a file extension for the toolbar item that creates a new file) to one of the following collections:
+
+- [items](/Documentation/ApiReference/UI_Widgets/dxFileManager/Configuration/toolbar/items/) - Displays toolbar items when no one file system item is selected.
```
Comment (dirk-pieterse) [inline api-reference/_hidden/dxFileManagerToolbar/items/items.md]: no one?

File: api-reference/_hidden/dxFileManagerToolbar/items/items.md [hunk 3]
```diff
@@ -23,30 +42,93 @@ The **FileManager** widget allows you to add default and create custom toolbar i
...
+                ]
+            }
+        });
+    });
+
+---
+
+**Custom Items**
+
+To add a custom toolbar item, specify its [text](/Documentation/ApiReference/UI_Widgets/dxFileManager/Configuration/toolbar/items/#text) and optional settings (for example, a file extension for the toolbar item that creates a new file) to one of the following collections:
```
Comment (dirk-pieterse) [inline api-reference/_hidden/dxFileManagerToolbar/items/items.md]: Same here

File: api-reference/_hidden/dxFileManagerToolbar/items/items.md [hunk 4]
```diff
@@ -23,30 +42,93 @@ The **FileManager** widget allows you to add default and create custom toolbar i
...
+
+- [items](/Documentation/ApiReference/UI_Widgets/dxFileManager/Configuration/toolbar/items/) - Displays toolbar items when no one file system item is selected.
+
+- [fileSelectionItems](/Documentation/ApiReference/UI_Widgets/dxFileManager/Configuration/toolbar/fileSelectionItems/) - Displays toolbar items when one or more file system items are selected.
+
+The [widget](/Documentation/ApiReference/UI_Widgets/dxFileManager/Configuration/toolbar/items/#widget) option allows you to specify a widget for a custom toolbar item ([dxButton](/Documentation/ApiReference/UI_Widgets/dxButton/) is used by default). Use the [toolbarItemClick](/Documentation/ApiReference/UI_Widgets/dxFileManager/Events/#toolbarItemClick) event to handle clicks on custom toolbar items.
```
Comment (dirk-pieterse) [inline api-reference/_hidden/dxFileManagerToolbar/items/items.md]: Why do you add "by default"?

**Review by dirk-pieterse — CHANGES_REQUESTED** [review]

File: api-reference/_hidden/dxFileManagerToolbar/items/items.md (hunk 1 above)
Comment (AnnaKondratova) [reply api-reference/_hidden/dxFileManagerToolbar/items/items.md]: updated

File: api-reference/_hidden/dxFileManagerToolbar/items/items.md (hunk 2 above)
Comment (AnnaKondratova) [reply api-reference/_hidden/dxFileManagerToolbar/items/items.md]: fixed

File: api-reference/_hidden/dxFileManagerToolbar/items/items.md (hunk 3 above)
Comment (AnnaKondratova) [reply api-reference/_hidden/dxFileManagerToolbar/items/items.md]: fixed

File: api-reference/_hidden/dxFileManagerToolbar/items/items.md (hunk 4 above)
Comment (AnnaKondratova) [reply api-reference/_hidden/dxFileManagerToolbar/items/items.md]: If you do not specify the widget option, the dxButton widget is used as default widget.

Comment (dirk-pieterse) [reply api-reference/_hidden/dxFileManagerToolbar/items/items.md]:
```suggestion
The [widget](/Documentation/ApiReference/UI_Widgets/dxFileManager/Configuration/toolbar/items/#widget) option allows you to specify a widget for a custom toolbar item ([dxButton](/Documentation/ApiReference/UI_Widgets/dxButton/) is the default widget). Use the [toolbarItemClick](/Documentation/ApiReference/UI_Widgets/dxFileManager/Events/#toolbarItemClick) event to handle clicks on custom toolbar items.
```

**Review by dirk-pieterse — APPROVED** [review]

**Pull request merged**